packages/foam-cli/src/utils/write-file-to-disk.ts

```diff
@@ -0,0 +1,8 @@
+import * as fs from 'fs';
+
+export const writeFileToDisk = async (fileUri: string, data: string): Promise<Boolean> => {
+  return fs.promises.writeFile(fileUri, data).then(() => true).catch(err => {
```

Comment:
Why do we catch and log here instead of letting the Error bubble up to the caller?

I'm not opposed to it, just wondering if it's necessary

Reply:
Done!